dhall-lsp-server/src/Dhall/LSP/Handlers.hs

```diff
@@ -337,10 +337,10 @@ documentFormattingHandler request = do
     _ -> throwE (Warning, "Failed to format dhall code; parse error.")
 
   ServerConfig {..} <- getServerConfig
-  let charSet | asciiOnly = ASCII
+  let characterSet | asciiOnly = ASCII
               | otherwise = Unicode
```

Comment:
```suggestion
  let characterSet
          | asciiOnly = ASCII
          | otherwise = Unicode
```

Same comment for the other two occurrences of this pattern

Reply:
is there an autoformatter?

Reply:
We do not currently use an autoformatter